eidpauth/src/test/java/uk/ac/ucl/eidp/auth/AuthServiceNgTest.java

```diff
@@ -0,0 +1,107 @@
+package uk.ac.ucl.eidp.auth;
+
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import uk.ac.ucl.eidp.auth.jwt.Jwt;
+import uk.ac.ucl.eidp.auth.model.RoleE;
+import uk.ac.ucl.eidp.auth.model.UserE;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Unit tests for AuthService.
+ * @author rebmdgu
+ */
+public class AuthServiceNgTest {
+  
+  private AuthService instance;
+  private final Set<RoleE> roleSet = new HashSet<>();
+  
+  public AuthServiceNgTest() {
```

Comment:
\[PMD\] INFO: Avoid unnecessary constructors \- the compiler will generate these for you